ompi/attribute/attribute.c

```diff
@@ -313,9 +353,11 @@ do { \
         /* MPI-2 Fortran-style */ \
         else { \
             MPI_Aint in, out;                        \
-            in = translate_to_fortran_mpi2(in_attr); \
-            (*((keyval_obj->copy_attr_fn).attr_mpi2_fortran_copy_fn)) \
-                (&(((ompi_##type##_t *)old_object)->attr_##type##_f), \
+            MPI_Fint attr_##type##_f;                \
```

Comment:
Oh, this is the keyval -- not the value.  Got it.  Disregard my question, then...